tasks/settings.yml

```diff
@@ -8,6 +8,15 @@
     state: present
   register: jenkins_init_prefix
 
+- name: Add Java options to Jenkins launch command
+  lineinfile:
+    dest: "{{ jenkins_init_file }}"
+    insertafter: '^JENKINS_JAVA_OPTIONS='
+    regexp: '^JENKINS_JAVA_OPTIONS=\"\$JENKINS_JAVA_OPTIONS '
+    line: 'JENKINS_JAVA_OPTIONS="$JENKINS_JAVA_OPTIONS {{ jenkins_java_options }}"'
```

Comment:
`jenkins_java_options` will also need to be added to the `defaults/main.yml` file (or you could add a `when: jenkins_java_options is defined` to this task).